docs/source/app_developers_guide/creating_sawtooth_network.rst

```diff
@@ -609,39 +710,486 @@ Confirm Network Functionality
+   KVM-QEMU, or Hyperkit. The steps in this procedure assume
+   `VirtualBox <https://www.virtualbox.org/wiki/Downloads>`_ (the default).
+
+#. Install the ``kubectl`` command as described in the Kubernetes document
+   `Install kubectl <https://kubernetes.io/docs/tasks/tools/install-kubectl/>`_.
+
+   * Linux quick reference:
+
+     .. code-block:: none
+
+        $ curl -Lo kubectl https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/amd64/kubectl \
+        && chmod +x kubectl && sudo cp kubectl /usr/local/bin/ && rm kubectl
+
+   * Mac quick reference:
+
+     .. code-block:: none
+
+        $ curl -Lo kubectl https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/darwin/amd64/kubectl \
+        && chmod +x kubectl && sudo cp kubectl /usr/local/bin/ && rm kubectl
+
+#. Install ``minikube`` as described in the Kubernetes document
+   `Install Minikube <https://kubernetes.io/docs/tasks/tools/install-minikube/>`_.
+
+   * Linux quick reference:
+
+     .. code-block:: none
+
+        $ curl -Lo minikube https://storage.googleapis.com/minikube/releases/latest/minikube-linux-amd64 \
+        && chmod +x minikube && sudo cp minikube /usr/local/bin/ && rm minikube
+
+   * Mac quick reference:
+
+     .. code-block:: none
+
+        $ curl -Lo minikube https://storage.googleapis.com/minikube/releases/latest/minikube-darwin-amd64 \
+        && chmod +x minikube && sudo mv minikube /usr/local/bin/
+
+
+Step 2: Start and Test Minikube
+-------------------------------
+
+This step summarizes the procedure to start Minikube and test basic
+functionality. If you have problems, see the Kubernetes document
+`Running Kubernetes Locally via Minikube
+<https://kubernetes.io/docs/setup/minikube/>`_.
+
+1. Start Minikube.
+
+   .. code-block:: console
+
+      $ minikube start
+
+#. Start Minikube's "Hello, World" test cluster, ``hello-minikube``.
+
+   .. code-block:: console
+
+      $ kubectl run hello-minikube --image=k8s.gcr.io/echoserver:1.10 --port=8080
+
+      $ kubectl expose deployment hello-minikube --type=NodePort
+
+#. Check the list of pods.
+
+   .. code-block:: console
+
+      $ kubectl get pods
+
+   After the pod is up and running, the output of this command should display a
+   pod starting with ``hello-minikube...``.
+
+#. Run a ``curl`` test to the cluster.
+
+   .. code-block:: none
+
+      $ curl $(minikube service hello-minikube --url)
+
+#. Remove the ``hello-minikube`` cluster.
+
+   .. code-block:: console
+
+      $ kubectl delete services hello-minikube
+
+      $ kubectl delete deployment hello-minikube
+
+
+Step 3: Download the Sawtooth Configuration File
+------------------------------------------------
+
+Download the Kubernetes configuration (kubeconfig) file
+``sawtooth-kubernetes-default-poet.yaml``.
```

Comment:
Another note for reviewers: This file is in PR 1829:
https://github.com/hyperledger/sawtooth-core/pull/1829